src/rgw/rgw_sal_motr.cc

```diff
@@ -1246,8 +1246,35 @@ MotrObject::~MotrObject() {
 
 int MotrObject::set_obj_attrs(const DoutPrefixProvider* dpp, RGWObjectCtx* rctx, Attrs* setattrs, Attrs* delattrs, optional_yield y, rgw_obj* target_obj)
```

Comment:
btw there is lot of code duplication from get_obj_attrs(), though given we are starting quite afresh on cortx-rgw, lets address that in future sprints, given we want to touch minimal code, without affecting any other functionality. 

Add a TODO to modularize code and use common code across get / set obj attr calls

Reply:
Done, Added